osu.Game/Skinning/Editor/SkinEditor.cs

```diff
@@ -57,13 +57,43 @@ private void load()
                 RelativeSizeAxes = Axes.Both,
                 Children = new Drawable[]
                 {
-                    headerText = new OsuTextFlowContainer
+                    new Container
                     {
-                        TextAnchor = Anchor.TopCentre,
-                        Padding = new MarginPadding(20),
-                        Anchor = Anchor.TopCentre,
-                        Origin = Anchor.TopCentre,
-                        RelativeSizeAxes = Axes.X
+                        Name = "Top bar",
+                        RelativeSizeAxes = Axes.X,
+                        Depth = float.MinValue,
+                        Height = 40,
+                        Children = new Drawable[]
+                        {
+                            new EditorMenuBar
+                            {
+                                Anchor = Anchor.CentreLeft,
+                                Origin = Anchor.CentreLeft,
+                                RelativeSizeAxes = Axes.Both,
+                                Items = new[]
+                                {
+                                    new MenuItem("File")
+                                    {
+                                        Items = new[]
+                                        {
+                                            new EditorMenuItem("Save", MenuItemType.Standard, Save),
+                                            new EditorMenuItem("Revert to default", MenuItemType.Destructive, revert),
```

Comment:
Just going to leave a drive-by remark that the revert to default option could probably use a confirmation dialog in the future. I'm bringing this up now because it is being moved here and the menu item is smaller and easier to fat-finger than the buttons used to be, but I don't really consider that a blocker for this PR as the dialog is also missing on `master` and adding it is out of scope of this pull. Something for a follow-up pass, probably.